src/components/common/TargetMap/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Map, GoogleApiWrapper, Marker } from 'google-maps-react';
+import './styles.scss';
+
+const mapStyles = {
+  width: '100%',
+  height: '100%',
+};
+
+const TargetMap = ({ position, targets, google, topics, onMapClicked }) => {
+  return (
+    <Map
+      zoom={14}
+      google={google}
+      style={mapStyles}
+      initialCenter={{
+        lat: position?.coords?.latitude,
+        lng: position?.coords?.longitude,
+      }}
+      onClick={onMapClicked}
+    >
+      <Marker
+        onClick={() => {}}
+        name={'Your Position'}
+        icon={{
+          url: '/current-position-marker.png',
+        }}
+      />
+      {targets.map(target => (
+        <Marker
+          position={{ lat: target.lat, lng: target.lng }}
```

Comment:
you can destructure here: {lat, lng} = target at the top and. then use ES6 shorthand and use position{{lat, lng}}

Reply:
Done, although if target keeps growing this may end up a little messy and will revert to the variable notation